introduction.md

```diff
@@ -0,0 +1 @@
+Hi there! I'm Mark Smit, a freelance senior C# .NET WebDeveloper. I'm checking out CodeSchool to see if there's interesting lessons there. Best regards, Mark
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.